<?xml version="1.0"?>
<launch>

<group ns="ur5">
      <node name="ati_bridge" pkg="soma_ur5" type="ati_bridge"/>
 </group>
 
</launch>